src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const RESULT = {};
+  const declarations = sourceString
```

Comment:
All chained methods called on the same level should have the same indentation to improve code readability.